Apps/Sandcastle/gallery/GeoJSON simplestyle.html

```diff
@@ -27,17 +27,23 @@
 function startup(Cesium) {
     "use strict";
 //Sandcastle_Begin
-var viewer = new Cesium.Viewer('cesiumContainer', {timeline : false, animation : false});
-
 //Load a GeoJSON file containing simplestyle information.
 //To learn more about simplestyle, see https://github.com/mapbox/simplestyle-spec
-viewer.dataSources.add(Cesium.GeoJsonDataSource.fromUrl('../../SampleData/simplestyles.geojson'));
-
-//Set the camera to view the data.
-viewer.camera.viewRectangle(Cesium.Rectangle.fromDegrees(-0.1, 0.2, 1.5, -1.1));
 
 //In this particular example, the name of each entity is set to its maki icon identifier.
 //Clicking on each billboard will show it's identifier in the InfoBox.
+
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    sceneMode : Cesium.SceneMode.SCENE2D,
+    timeline : false,
+    animation : false
+});
+
+var dataSource = new Cesium.GeoJsonDataSource();
+viewer.dataSources.add(dataSource);
+dataSource.loadUrl('../../SampleData/simplestyles.geojson').then(function(){
```

Comment:
I guess technically it's all feasible, it just goes against the ideals we've been maintaining in our architecture.  In order to support any of your suggestions, the entity would have to know what viewer/scene it is currently in. The entire DataSource layer is built around separation of concerns, entities and data sources describe the data and then the visualizers and data source display handle the actual visualization of that data. This architecture has a number of benefits and if I were building an application out of Cesium, I would want to keep that separation intact.

I would like to just stick with the `viewer` functions for now and then we can revisit this post 1.6 after hearing what users think of it.  I don't want to start trying to do too much too fast without getting more feedback (not to mention we're already up to 4000 new lines of code with this whole effort).